mirage/config.js

```diff
@@ -46,6 +46,18 @@ export default function () {
     }));
   });
 
+  this.put('/vendors/:vendorId/packages/:packageId/titles/:titleId', ({ customerResources, titles }, request) => {
+    const matchingCustomerResource = customerResources.findBy({
```

Comment:
Let's use `let` or `const`, but not both for intermediate computations.

My vote is that we use `const` for "true" constants: PI, some URL, etc.... that live at the top-level of a module, but for values that are intermediate to a computation, we use `let`

I am, of course, biased in this matter. So much so, that I wrote an eslint plugin https://www.npmjs.com/package/eslint-plugin-prefer-let

However, I'm more committed to consistency.

Reply:
I'm ok with restricting `const` to the top-level. My own thoughts on it are: "use `const` if you are not going to change this value later, otherwise use `let`"